### PR TITLE
Security upgrades to django, djangorestframework, jinja2, pillow

### PR DESCRIPTION
**Pull request by haavikko**
## Description :sparkles:

NOTE: Major version upgrade to Pillow. We are not using any pillow functionality directly. Is it only used for the defunct map file upload?

### Closes :no_good_woman:
**[VEN-XXX](https://helsinkisolutionoffice.atlassian.net/browse/VEN-XXX):**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -44,11 +44,11 @@ graphql-core==2.3.2       # via django-graphql-jwt, django-ilmoitin, graphene, g
 graphql-relay==2.0.1      # via django-ilmoitin, graphene
 idna==2.10                # via requests
 itypes==1.2.0             # via coreapi
-jinja2==2.11.2            # via coreschema, django-ilmoitin, django-silk
+jinja2==2.11.3            # via coreschema, django-ilmoitin, django-silk
 json-log-formatter==0.3.0  # via -r requirements.in
 lockfile==0.12.2          # via django-mailer
 markupsafe==1.1.1         # via jinja2
-pillow==7.2.0             # via -r requirements.in
+pillow==8.1.2             # via -r requirements.in
```
Comment (a-rmz) [inline requirements.txt:51]: It seems like we're not even using it anymore 🤔

Comment (haavikko) [reply requirements.txt:51]: All the better then, I'll remove it completely

Comment (haavikko) [reply requirements.txt]: fixed

**Review by a-rmz — APPROVED** [review]

**Pull request merged**